src/Castle.Windsor.Tests/Diagnostics/DuplicatedDependenciesDiagnosticTestCase.cs

```diff
@@ -31,9 +31,6 @@ public class DuplicatedDependenciesDiagnosticTestCase : AbstractContainerTestCas
 		protected override void AfterContainerCreated()
 		{
 			var host = Kernel.GetSubSystem(SubSystemConstants.DiagnosticsKey) as IDiagnosticsHost;
-#if SILVERLIGHT
-			host.AddDiagnostic<IDuplicatedDependenciesDiagnostic>(new DuplicatedDependenciesDiagnostic(Kernel));
-#endif
```

Comment:
Ha! @jonorossi - join the club! Midnight owl coding takes it's toll on everyone :) :) :)